src/styles/themes/default.js

```diff
@@ -14,6 +14,7 @@ export default {
     border: colors.gray2,
     borderFocus: colors.blue,
     icon: colors.gray3,
+    iconLight: 'rgba(198, 201, 202, 0.5)',
```

Comment:
Instead of doing this, we can stick to the existing colors.gray values and pass the style as `opacity: 0.5`

Reply:
If we apply the `opacity` style to the text, it will affect the background color and the text itself. I was thinking that we could take the `icon` color and append the string with '80' (the hex will be #C6C9CA80), which is the hex code for the 50% opacity. So, it will look like this 
`StyleUtils.getBackgroundColorStyle(themeColors.icon + '80')`
but I think it looks ugly, that's why I decided to add a new color.

If we still want to apply the `opacity` style, I think we need to change the `TextPill` component a bit by wrapping the text with `View`.

What do you think?

Reply:
I am fine with wrapping the `View` around `Text`.

Reply:
Okay, so it turns out that wrapping it with `View` and apply the opacity to the `View` also affects the `Text`. So, I think the only way is to apply the opacity to the `backgroundColor`.

Reply:
Should I keep the `iconLight` color?

Reply:
I agree with the concern about opacity. I think it's fine to add a new color value here, but let's make sure it's a color in our variables/theme files. 

I think it would be something like `pillBGHighlighted` or something like that. But then let's reference the icon color here but change the opacity. I'm not sure how this is done in RN, but in Sass we might write something like:

```
$pillBGHighlighted: transparentize($color-icon, 50%);
```

Let me know if that makes sense.

Reply:
As far as I know, RN does not have such thing, but we can make our own implementation. I am thinking that maybe we could make a new function in `StyleUtils.js` let say `getBackgroundColorWithOpacityStyle` which takes the hex code as string and the opacity as number (0 to 1). Then, we will convert that hex to `rgb` numbers then return `rgba(r, g, b, opacity)`. It will look like this:
```
function getBackgroundColorWithOpacityStyle(background, opacity) {
    const r = ....
    const g = ....
    const b = .....
    return {
        backgroundColor: `rgba(${r}, ${g}, ${b}, ${opacity})`
    }
}
```

What do you think?

Reply:
I will lean on @mananjadhav and @marcochavezf to evaluate that, but I like the idea.

Reply:
@bernhardoj Can we not just use `rgb(r, g, b, 0.5)` in the `colors.js` ?

Reply:
Yes, we can remove the `iconLight` and use the new function above that I propose like this:
`StyleUtils.getBackgroundColorWithOpacityStyle(themeColors.icon, 0.5)`.

Reply:
My newest commit currently uses above approach. Let me know if there is any concern.